wfe2/wfe_test.go

```diff
@@ -3228,7 +3228,7 @@ func TestPrepAuthzForDisplay(t *testing.T) {
 		Identifier:     identifier.DNSIdentifier("*.example.com"),
 		Challenges: []core.Challenge{
 			{
-				Type: "dns",
+				Type:                     "dns",
```

Comment:
Omg, I'm so happy I don't have to sed this to pass the 1.18 lints anymore!

```suggestion
				Type:                     "dns",
```

Reply:
SAME